artemis-tools/src/test/resources/artemis-configuration.xsd

```diff
@@ -3456,6 +3456,14 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="enable-ingress-timestamp" type="xsd:boolean" default="true" maxOccurs="1" minOccurs="0">
```

Comment:
Similar to default value comment for earlier xsd file (this one looks test-only so may not matter though)